test/e2e/autoscaling_utils.go

```diff
@@ -60,9 +60,10 @@ type ResourceConsumer struct {
 NewResourceConsumer creates new ResourceConsumer
 initCPU argument is in millicores
 initMemory argument is in megabytes
+memLimit argument is in megabytes
```

Comment:
Please add comment about cpuLimit as well. Please also add comment that both cpu and mem limit are the maximum amount of resources you expect to consume for a single pod. The situation when you specify replicas=1, initMemory=256M, memLimit=256M and then increase the memory consumption to 512M might be confusing without exhaustive explanation of what cpu/mem limit is.